README.md

```diff
@@ -40,6 +40,13 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
    ./generate-yaml.sh [options] <path/to/clouds.yaml> <openstack cloud> <provider os>
    cd ../..
    ```
+
+   `<clouds.yaml>` is a yaml file to record how to interact with openstack cloud, there's a sample
+   [clouds.yaml](pkg/cloud/openstack/clients/clouds.yaml), and https://docs.openstack.org/python-openstackclient/latest/configuration/index.html#configuration-files has additional information.
+
+   `<openstack cloud>` is the cloud you are going to use, e.g multiple cloud might be defined in clouds.yaml
```

Comment:
Can you please show more detail for what is the content for `<openstack cloud>` in an example?